### DISCUSSION TITLE
Is it possible to Track-only html (get version by text)?

### DISCUSSION THREAD

**Discussion by yaudahj** [ImranR98/Obtainium#2010] [Q&A]
Well I'm new to this app and I'm not sure how to make a config while figuring out how this app work. However this is the case: Since the web using JavaScript to get the download link, while this app unable to process it, I'm planned to use as update notification so I can download manually after get notified. However I don't know how to do it. Please any help...

> If you asking of me what's the website: [LITEAPKS](https://liteapks.com/download/ircode-finder-nec-protocol-69974)

**Comment by DwainZwerg** [comment]
See https://github.com/ImranR98/Obtainium/issues/1239
